apps/judicial-system/web/src/routes/Shared/Requests/Requests.tsx

```diff
@@ -80,7 +80,7 @@ export const Requests: React.FC = () => {
   const resCases = data?.cases
 
   useEffect(() => {
-    document.title = 'Allar kröfur - Réttarvörslugátt'
```

Comment:
We have a separate task to add this to CF: https://app.asana.com/0/1199153462262248/1201934486933573/f